Rspec/spec/solver_spec.rb

```diff
@@ -0,0 +1,32 @@
+require_relative './../../solver'
+
+describe Solver do
+  context 'When testing the Solver class' do
+    it 'should return the factorial of number passed to the factorial method of class Solver' do
+      solver = Solver.new
+      expect(solver.factorial(0)).to eq 1
+      expect(solver.factorial(1)).to eq 1
+      expect(solver.factorial(5)).to eq 120
```

Comment:
- [ ] As requested by the previous reviewer, please write tests for the factorial method which should **raise an exception when a negative value is passed in**

As a hint, the assertion stage should be something like the below 👇
```ruby 
expect ( solver.factorial(-5) ).to raise_exception
```

Remember to write that condition too 💡 😉